src/platforms/android.ts

```diff
@@ -11,6 +11,20 @@ import {
 } from "../helpers.js";
 import { Platform, uniformIconOptions } from "./base.js";
 
+interface Icon {
+  readonly src: string;
+  readonly sizes: string;
+  readonly type: string;
+}
+
+interface Shortcut {
+  readonly name: string;
+  readonly short_name: string;
+  description?: string;
```

Comment:
```suggestion
  readonly description?: string;
```

Reply:
done
I needed to do a little rewrite for that though.